docs/axes/cartesian/time.md

```diff
@@ -9,7 +9,7 @@ The time scale is used to display times and dates. When building its ticks, it w
 The x-axis data points may additionally be specified via the `t` attribute when using the time scale.
 
     data: [{
-        x: new Date(),
```

Comment:
I think the idea here was to show one datapoint with `x` and one with `t` to demonstrate that they both work

Reply:
ok, I have let the datapoint with x and have changed only the previous instruction.